Source/Interceptor.cs

```diff
@@ -131,10 +131,8 @@ public void AddCall(IProxyCall call)
 			new Lazy<IInterceptStrategy[]>(
 				() => new IInterceptStrategy[]
 				{
-					HandleFinalizer.Instance,
 					HandleTracking.Instance,
-					InterceptMockPropertyMixin.Instance,
-					InterceptObjectMethodsMixin.Instance,
+					HandleWellKnownMethods.Instance,
```

Comment:
Handling finalizers after `HandleTracking` *might* in theory be problematic since the objects related to the one being finalized might already be gone. But then, assuming that the underlying proxy library might intercept a finalizer is just as problematic. (DynamicProxy doesn't intercept finalizers, so all logic in Moq that deals with a finalizer call is really just a backup.)